components/openindiana/ddu/ddu-library.p5m

```diff
@@ -51,9 +51,9 @@ file path=usr/ddu/scripts/pkg_relate.sh
 file path=usr/ddu/scripts/probe.sh
 file path=usr/ddu/scripts/url_down.py
 file path=usr/ddu/scripts/wireless_NIC_keywords
-file path=usr/lib/python3.5/vendor-packages/DDU/__init__.py
-file path=usr/lib/python3.5/vendor-packages/DDU/ddu_devdata.py
-file path=usr/lib/python3.5/vendor-packages/DDU/ddu_errors.py
-file path=usr/lib/python3.5/vendor-packages/DDU/ddu_function.py
-file path=usr/lib/python3.5/vendor-packages/DDU/ddu_package.py
-file path=usr/lib/python3.5/vendor-packages/DDU/ddu_repo.py
+file path=usr/lib/python3.9/vendor-packages/DDU/__init__.py
```

Comment:
I suggest to replace `3.9` by `$(PYVER)` so we do not need to change this file on every Python upgrade.

Reply:
Done